AIPM-Registry/DATA_INGESTION_PROMPT.md

```
@@ -0,0 +1,402 @@
+
+- #
+
+# 1) Find All — Entity Discovery (datasets
+
+)
+
+**Goal**: Enumerate entities across the AI landscape for the AI Standardization Registry
+
+.
+
+**Entity types to discover**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)